keras/src/backend/config.py

```diff
@@ -403,3 +438,13 @@ def max_steps_per_epoch():
             `None`, no limit is applied.
     """
     return _MAX_STEPS_PER_EPOCH
+
+
+if "KERAS_NNX_ENABLED" in os.environ:
+    env_val = os.environ["KERAS_NNX_ENABLED"].lower()
+    if env_val:
```

Comment:
This seems bad, don't we want to check if val.lower() = "true" or "1" here? Otherwise KERAS_NNX_ENABLED=False will enable nnx.

Reply:
done